src/components/learning-units-section/complete-learning-unit-toogle/CompleteLearningUnitToggleNoAuth.js

```diff
@@ -0,0 +1,18 @@
+import { CompleteLearningUnitToggleButton } from './CompleteLearningUnitToggleButton';
+import { Button } from 'primereact/button';
+import style from './CompleteLearningUnitToogle.module.scss';
+import useLoginDialog from '@hooks/useLoginDialog';
+
+export const CompleteLearningUnitToggleNoAuth = () => {
+  const loginDialog = useLoginDialog();
+  return (
+    <div>
+      <Button
+        className={`p-button-text p-button-plain ${style.button_without_style}`}
+        onMouseEnter={() => loginDialog.setDisplayLoginDialog(true)}
+      >
+        <CompleteLearningUnitToggleButton className="p-disabled" />
```

Comment:
here i think you could make a prop for the component like this:
`<CompleteLearningUnitToggleButton disabled={true} />` and then manage it inside the component

Reply:
The component CompleteLearningUnitToggleButton could be deleted and use directly the component ToggleButton from primefaces.

Reply:
Done using Carlos suggestion.